system/lib/pthread/thread_profiler.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <string.h>
+#include "pthread_impl.h"
+#include "emscripten/threading.h"
+// Included for emscripten_builtin_free / emscripten_builtin_malloc
+// TODO(sbc): Should these be in their own header to avoid emmalloc here?
+#include "emscripten/emmalloc.h"
```

Comment:
nit: use the angle brackets include style here to prevent the preprocessor from searching in the same directory first.
```suggestion
#include <emscripten/threading.h>
// Included for emscripten_builtin_free / emscripten_builtin_malloc
// TODO(sbc): Should these be in their own header to avoid emmalloc here?
#include <emscripten/emmalloc.h>
```